master/docs/developer/secrets.rst

```diff
@@ -1,6 +1,10 @@
 Secrets
 -------
 
+.. spelling::
+
+  secretDetails
```

Comment:
Please wrap `secretDetails` in double backticks instead --- it's code, not English sentence:

```
return a list of ``secretDetails``.
```